.github/workflows/maven-publish.yml

```diff
@@ -0,0 +1,35 @@
+name: Publish snapshots to maven
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - 'main'
+      - '1.*'
```

Comment:
This task is not present for 1.x version. @prudhvigodithi  can confirm but it can be removed later is required.

Reply:
Hey @gaiksaya you are right it should be starting from 2.x.